ilastik/applets/networkClassification/nnClassGui.py

```diff
@@ -157,10 +159,9 @@ def __init__(self, parent):
         super(QDialog, self).__init__(parent=parent)
 
         self.validation_size = QComboBox(self)
-        self.validation_size.addItem('10%')
-        self.validation_size.addItem('20%')
-        self.validation_size.addItem('30%')
-        self.validation_size.addItem('40%')
+        self.validation_size.addItem('1')
```

Comment:
Maybe use `QSpinBox` and allow users to set any value they want (within a "reasonable" range)?